frontend/src/views/CellGuide/components/CellGuideCard/index.tsx

```diff
@@ -216,6 +221,49 @@ export default function CellGuideCard({
     setSelectedGene(undefined);
   }, [selectedOrgan, selectedOrganism, setSelectedGene]);
 
+  const { computationalMarkerGeneTableData } =
+    useComputationalMarkerGenesTableRowsAndFilters({
+      cellTypeId,
+      organismName: selectedOrganism.name,
+      organId: selectedOrganId,
+    });
+
+  const sdsGenesList = useMemo(() => {
+    const genes = computationalMarkerGeneTableData.map((gene) => ({
+      name: gene.symbol,
+    }));
+
+    // Insert an element at the beginning
+    genes.unshift({ name: "No Gene" });
+
+    return genes;
```

Comment:
let's do the following for clarity 🙏 

Reading the code, it's kinda jarring to see an unshift to insert an element to the front unnecessarily

```ts
const NO_GENE_OPTIONS = { name: "No Gene" };

return [NO_GENE_OPTIONS, ... genes];
```